recipes/usalign/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "USalign" %}
+{% set version = "20240303" %}
+{% set sha256 = "b4da9fc58449f96a98eabf4137082cfc161787a5df739df351186e70b9c87950" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://zhanggroup.org/US-align/bin/module/USalignLinux64.zip
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64  # [linux]
+
+test:
+  commands:
+    - which USalign
+
+about:
+  home: https://zhanggroup.org/US-align
+  summary: 'US-align: universal structure alignment of monomeric and complex proteins and nucleic acids'
+  description: |
+    US-align (Universal Structural alignment) is a unified protocol
+    to compare 3D structures of different macromolecules (proteins, RNAs and DNAs)
+    in different forms (monomers, oligomers and heterocomplexes) for
+    both pairwise and multiple structure alignments.
+  license: BSD-2-Clause
+  license_family: BSD
```

Comment:
[This is not BSD-2-Clause](https://opensource.org/license/bsd-2-clause). Use "OTHER" for the license description or figure out what the license actually is.